src/web100srv.c

```diff
@@ -927,28 +802,32 @@ int run_test(tcp_stat_agent* agent, int ctlsockfd, TestOptions* testopt,
 
   time_t stime;
 
-  double rttsec;  // average round trip time
+  double rttsec;      // average round trip time
   double swin, cwin;  // send, congestion window sizes respectively
-  double rwin;  // max window size advertisement rcvd
-  double rwintime;  // time spent being limited due to rcvr end
-  double cwndtime;  // time spent being limited due to congestion
-  double sendtime;  // time spent being limited due to sender's own fault
+  double rwin;        // max window size advertisement rcvd
+  double rwintime;    // time spent being limited due to rcvr end
+  double cwndtime;    // time spent being limited due to congestion
+  double sendtime;    // time spent being limited due to sender's own fault
 
   double oo_order;  // out-of-order packet ratio
   double waitsec;
-  double bw_theortcl;  // theoretical bandwidth
-  double avgrtt;  // Average round-trip time
-  double timesec;  // Total test time in microseconds
+  double bw_theortcl;     // theoretical bandwidth
+  double avgrtt;          // Average round-trip time
+  double timesec;         // Total test time in microseconds
   double packetloss_s2c;  // Packet loss as calculated from S->c tests.
-  double RTOidle;  // Proportion of idle time spent waiting for packets
-  double s2cspd;  // average throughput as calculated by S->C test
-  double c2sspd;  // average throughput as calculated by C->S test
-  double s2c2spd;  // average throughput as calculated by midbox test
-  double realthruput;  // total send throughput in S->C
+  double RTOidle;         // Proportion of idle time spent waiting for packets
+  double s2cspd;          // average throughput as calculated by S->C test
+  double c2sspd;          // average throughput as calculated by C->S test
+  double s2c2spd;         // average throughput as calculated by midbox test
+  double realthruput;     // total send throughput in S->C
+  double acksratio;       // ratio of acks over packets sent
   double aspd = 0;
+  double tmoutsratio;  // timeouts fraction
+  // ratio of retransmissions and duplicate acks over packets sent
+  double rtranratio, dackratio;
```

Comment:
Seems strange to combine just these two variables into a single line when all others are on separate lines. If we're going to pair it with anything, it seems like `dackratio` and `acksratio` are more closely linked. Also suggest renaming to `dacksratio` for consistency.